src/features/people/peopleList/PersonTile/styled.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    padding: 16px;
+    background-color: ${({ theme }) => theme.colors.white};
+    box-shadow: 0px 4px 12px ${({ theme }) => theme.colors.tileBoxShadow};
+    border-radius: 5px;
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+    height: 100%;
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+        padding: 8px;
+    }
+`;
+
+export const Poster = styled.img`
```

Comment:
same as above, Poster from MovieTile is similar

Reply:
we will discuss this